packages/components/src/radio/docs/radiogroup/description.tsx

```diff
@@ -1,16 +1,13 @@
-import { HelperMessage, RadioField, RadioGroup, Radio, RadioList } from "@hopper-ui/components";
+import { Radio, RadioField, RadioGroup } from "@hopper-ui/components";
 
 export default function Example() {
     return (
-        <RadioGroup aria-label="roles">
-            <RadioList>
-                <Radio value="developer">Developer</Radio>
-                <Radio value="designer">Designer</Radio>
-                <RadioField>
-                    <Radio value="manager">Manager</Radio>
-                </RadioField>
-            </RadioList>
-            <HelperMessage>Select one to continue</HelperMessage>
+        <RadioGroup aria-label="roles" description="Select one to continue">
+            <Radio value="developer">Developer</Radio>
+            <Radio value="designer">Designer</Radio>
+            <RadioField>
```

Comment:
[Question] What's the use of an empty RadioField?

Reply:
hmm good point, I probably forgot to add the description. Good catch!

Reply:
after looking at it, it's not even needed here so i removed it.